### DISCUSSION TITLE
Kurs aus CP raus rückwärts starten

### DISCUSSION THREAD

**Discussion by BennyClavis** [Courseplay/Courseplay_FS22#2366] [Ideas]
Guten Tag, wäre es möglich eine Option einzubauen, dass ein weiterer Fahrer den erstellten Kurs entgegengesetzt abzufahren, ich mochte gern nach dem Heu machen mit Schwad, einen 2 Fahrer entgegengesetzt zum normalen Abfahrer starten lassen, es gibt zwar die Möglichkeit, dass ich 2 Wender auf der Wiese das Heu zum Schwad machen lassen kann und dann 2 Ladewagen den Kurs hinterher abfahren lasse, aber ich fände es schön, wenn ich einen den Kurs entgegengesetzt starten lassen könnte. MfG Benny

**Comment by Tensuko** [comment]
Sowas war mal angedacht ja. Müssen wir mal schauen ob wir das umsetzen.
Es gehört eben noch etwas mehr dazu also nur die Wegpunkte in anderer Reihnfolge zu laden.
